R/check-input-helpers.R

```diff
@@ -210,20 +233,20 @@ check_columns_present <- function(data, columns) {
   assert_character(columns, min.len = 1)
   colnames <- colnames(data)
   missing <- list()
-  for (x in columns){
+  for (x in columns) {
     if (!(x %in% colnames)) {
       missing[[x]] <- x
     }
   }
   missing <- unlist(missing)
-  if (length(missing) > 1) {
-    msg <- paste0(
-      "Columns '", paste(missing, collapse = "', '"), "' not found in data"
+  if (length(missing) >= 1) {
+    return(
+      cli_inform(
```

Comment:
I think this should not return `cli_inform`. As is, the function immediately outputs a message to the console. Previously, the function returned a string with a message that would then be handled by the function calling `check_columns_present()`. 

With the current updated behaviour, you always get a message when calling `as_forecast()` about either `sample_id` or `quantile_level` missing.

Reply:
I think the latest version is the correct version!
This is marked as outdated and I think if you click on it you get to the original version I made a comment for.